http/vibe/http/websockets.d

```diff
@@ -567,8 +680,18 @@ final class WebSocket {
 	*/
 	void close(short code = 0, scope const(char)[] reason = "")
 	{
+		//assume the default is normal, intentional closure
+		if(code == 0)
+			code = WebSocketCloseReason.normalClosure;
+
+		if(reason is null || reason.length == 0)
+			reason = (cast(WebSocketCloseReason)code).toString;
+
 		//control frame payloads are limited to 125 bytes
-		assert(reason.length <= 123);
+		version(assert)
+			assert(reason.length <= 123);
+		else
+			reason = reason[0 .. 123];
```

Comment:
Should be `reason = reason[0 .. min($, 123)];` to avoid a range error.

Reply:
Thanks for this catch!